procfile/README.md

```diff
@@ -42,3 +42,13 @@ Specifies the command that should be run when executing this process
 ```yaml
 command: ./bin/web
 ```
+
+**Expose**
+
+Specifies the exposure settings for the process. This allows you to expose a tcp/http/https/ssl service on any process in the application.
+
+```yaml
+expose:
+  type: tcp // Possible options are tcp/http/https/ssl
```

Comment:
Just tested this with ECS. It looks like it's possible, but their docs don't make it very intuitive. When we create an ECS, we provide an array of [ecs.LoadBalancer](http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_LoadBalancer.html). That makes you provide a ContainerPort and ContainerName, but those don't seem to serve any purpose other than validation. So, if we create a load balancer with multiple listeners, and a task definition with multiple port mappings, that should be fine.